### PR TITLE
UX-934/ Create RBAC profile

### PR DESCRIPTION
**Pull request by xantorres**

### CODE REVIEW

File: src/app/plugins/kubernetes/components/rbac/actions.js [hunk 1]
```diff
@@ -391,3 +391,34 @@ export const clusterRoleBindingActions = createCRUDActions(ActionDataKeys.Cluste
...
+      kind: 'ClusterProfile',
+      metadata: { name: data.profileName },
+      spec: {
+        cloneFrom: data.baseCluster,
+        namespaceScopedResources: [...roles, ...roleBindings],
+        clusterScopedResources: [...clusterRoles, ...clusterRoleBindings],
+        reapInterval: 30,
+      },
+    }
+    Bugsnag.leaveBreadcrumb('Attempting to create a rbac profile', body)
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/rbac/actions.js:418]: lets also track this in analytics

File: src/app/plugins/kubernetes/components/rbac/create-profile/role-bindings-step.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      <FormFieldCard
+        className={classes.listBox}
+        title={'RBAC Profile - Role Bindings'}
+        link={<SimpleLink src={profilesHelpUrl}>Profiles Help</SimpleLink>}
+      >
+        <RoleBindingsListTable
+          filters={
+            <NamespacePicklist
+              /*
+              // @ts-ignore */
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/rbac/create-profile/role-bindings-step.tsx:91]: i believe you can define a type of useParams so it recognizes these values

**Review by jruddell — APPROVED** [review]

File: src/app/plugins/kubernetes/components/rbac/create-profile/base-cluster-step.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import { createUsePrefParamsHook } from 'core/hooks/useParams'
+import ProfileSummaryBox from 'k8s/components/rbac/create-profile/profile-summary-box'
+import useStyles from './useStyles'
+
+const profilesHelpUrl = `${appUrlRoot}/help`
+const RbacProfileBox = ({ profileName, setWizardContext }) => {
+  const classes = useStyles()
+  return (
+    <FormFieldCard
+      title={'RBAC Profile'}
```
Comment (kevinylee1991) [inline src/app/plugins/kubernetes/components/rbac/create-profile/base-cluster-step.tsx]: could just be title="RBAC Profile"

File: src/app/plugins/kubernetes/components/rbac/create-profile/base-cluster-step.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      link={<SimpleLink src={profilesHelpUrl}>Profiles Help</SimpleLink>}
+    >
+      <p>
+        RBAC Profiles allow you to snapshot a cluster and create 'profile' or template of the
```
Comment (kevinylee1991) [inline src/app/plugins/kubernetes/components/rbac/create-profile/base-cluster-step.tsx]: grammar: and create _**a**_ ''profile'

**Pull request merged**